src/windows.rs

```diff
@@ -21,6 +21,12 @@ extern "system" {
     ) -> u32;
 }
 
+extern "system" {
+    // Forbidden when targetting UWP
```

Comment:
Instead of a comment, we should have a `#[cfg(not(target_vendor = "uwp"))]` attribute. Something like:
```rust
#[link(name = "bcrypt")]
extern "system" {
    fn BCryptGenRandom(
        hAlgorithm: *mut c_void,
        pBuffer: *mut u8,
        cbBuffer: u32,
        dwFlags: u32,
    ) -> i32;
}

#[cfg(not(target_vendor = "uwp"))]
#[link(name = "advapi32")]
extern "system" {
  #[link_name = "SystemFunction036"]
  pub fn RtlGenRandom(RandomBuffer: *mut c_void, RandomBufferLength: u32) -> u8;
}
```

Note that the buffer is of type `PVOID` and we need to make sure we link to `Advapi32`